actors/Weapons/Slot4/MINIGUN.dec

```diff
@@ -622,7 +622,13 @@ States
 			A_FireBullets(4, 4, 1, 10, "MachineGunBulletPuff", 1);
 			A_FireCustomMissile("GunFireSmoke", 0, 0, 0, 0, 0, 0);
 			A_Firecustommissile("FiftyCaseSpawn",0,0,-12,-18);
-			A_FireCustomMissile("MinigunTracer", random(-3,3), 0, -1, random(-3,3));
+			if(GetCvar("PB_alttracer") >=1)
+				{
+				 A_RailAttack(0, 0, 0, none,none, RGF_SILENT|RGF_NOPIERCING|RGF_EXPLICITANGLE, 0, none,0.1,0.1,700,0,1,1.0,"Tracer_Trail",-6,0,0);
+				// A_FireBullets (0.1, 0.1, -1, 22, "HitPuff", FBF_NORANDOM,8192,none, -2,0);
+				}
+			else {A_FireCustomMissile("MinigunTracer", random(-3,3), 0, -1, random(-3,3));}
+			
```

Comment:
Is there a way to have this section defined once and then have other places in the code jump to it instead of copy/pasting it several times over? I'm thinking Goto-statements

Reply:
Yes I have done that with my own version of the minigun.  I can show you on discord if you like.  Long story short using jumps to an action and then knows what action it was previously at makes things much more complicated and would use a ton more code than here.
The only way for your suggestion to be lighter on code theoretically would be if you had a zscript function that spilled out that command when called.  I have no clue how to do program that though to be honest.

I could try code similar to the new rocket launcher and call upon that code using an overlay instead.  That way the decision is made on the tracer type within the overlay once and you simply keep calling it.  If I have time and am successful I can delete this PR and make a new one.